src/app.module.ts

```diff
@@ -12,15 +11,15 @@ import * as process from 'process';
     TypeOrmModule.forRoot({
       type: 'mysql',
       host: process.env.HOST,
-      port: 3306,
+      port: Number(process.env.PORT),
```

Comment:
This is not a PR blocker, asking for my own knowledge, is it required to have this as a number? a string will cause a type error for TypeOrm?